packages/nibijs/src/tx/signingClient.ts

```diff
@@ -108,3 +108,5 @@ export class NibiruSigningClient extends SigningStargateClient {
     }
   }
 }
+
+export { StdFee } from "@cosmjs/amino/build"
```

Comment:
Can we not export third-party packages/functions into our index barrels?
Would prefer directly importing the package into the file

Reply:
Fixed